pmc/test_procedures/default.json

```diff
@@ -70,92 +71,50 @@
         {
           "operation": "default",
           "warmup-iterations": 500,
-          "iterations": 200
-          {%- if not target_throughput %}
-          ,"target-throughput": 20
-          {%- elif target_throughput is string and target_throughput.lower() == 'none' %}
-          {%- else %}
-          ,"target-throughput": {{ target_throughput | tojson }}
-          {%- endif %}
-          {%- if search_clients is defined and search_clients %}
-          ,"clients": {{ search_clients | tojson}}
-          {%- endif %}
+          "iterations": 200,
+          "target-throughput": {{ target_throughput | default(20) | tojson }},
```

Comment:
Much cleaner. Thanks!

Reply:
Yes, using "0" is the same as the erstwhile "none".